HTML/QuickForm/file.php

```diff
@@ -320,10 +320,12 @@ static function _ruleCheckFileName($elementValue, $regex)
     * Needs to be redefined here as $_FILES is populated differently from
     * other arrays when element name is of the form foo[bar]
     *
+    * @param bool $sc1   unused, for signature compatibility
+    *
     * @access    private
     * @return    mixed
     */
-    function _findValue(&$values)
+    function _findValue(&$sc1 = null)
```

Comment:
change 7

Reply:
Did we miss a trick here - it seems less compatible with the parent

```
    /**
     * Tries to find the element value from the values array
     *
     * @since     2.7
     * @access    private
     * @return    mixed
     */
    function _findValue(&$values)
    {
```

Reply:
Not sure that is the change but this is the direct comparison from 3.2.11 to 3.2.16 of the upstream https://gist.github.com/seamuslee001/0a67ad08359f46b183a7e26f05c99658#file-quickform_diff-diff-L748